### ISSUE TITLE
phoneNumberValidator method can't accept phone numbers that start with '0900''

### ISSUE THREAD

**Issue by arezooalipanah** [ali-master/persian-tools#280]
HI
phoneNumberValidator method can't accept phone numbers that startswith '0900''

**Comment by TrueMoein** [comment]
+1

**Comment by TrueMoein** [comment]
https://900.ir/

**Comment by ali-master** [comment]
> HI phoneNumberValidator method can't accept phone numbers that startswith '0900''

The issue was just fixed in V3.4.1 and published. please check it out and let me know if the problem is fixed or still exists.

**Comment by arezooalipanah** [comment]
> > HI phoneNumberValidator method can't accept phone numbers that startswith '0900''
> 
> The issue was just fixed in V3.4.1 and published. please check it out and let me know if the problem is fixed or still exists.

tnx ali-master
it was ok

**Comment by mediv0** [comment]
> > HI phoneNumberValidator method can't accept phone numbers that startswith '0900''
> 
> The issue was just fixed in V3.4.1 and published. please check it out and let me know if the problem is fixed or still exists.

https://github.com/persian-tools/persian-tools/blob/13c3d20418732b7614661e5aadcecca4e6cb1ce1/src/modules/phoneNumber/normalizer.ts#L25-L26

Regex need also to be updated for #184

**Comment by geek-sajjad** [comment]
This issue is still open?

**Comment by ali-master** [comment]
> This issue is still open?

We appreciate it if someone implements the feature and fixes the issue.

**Comment by ShahriarKh** [comment]
How about this regex:
```js
/^\+?(0|00)?(98)?(9\d{9})$/
```
It accepts:
```
9123030300
09123030300
009123030300
989123030300
0989123030300
00989123030300
+989123030300
+0989123030300
+00989123030300
```

**Comment by mediv0** [comment]
> How about this regex:
> 
> ```js
> /^\+?(0|00)?(98)?(9\d{9})$/
> ```
> 
> It accepts:
> 
> ```
> 9123030300
> 09123030300
> 009123030300
> 989123030300
> 0989123030300
> 00989123030300
> +989123030300
> +0989123030300
> +00989123030300
> ```

already fixed in #283

**Comment by ali-master** [comment]
New version is coming soon.
